etc/kayobe/kolla.yml

```diff
@@ -180,6 +180,9 @@ stackhpc_rocky_9_additional_repos:
   - url: "{{ stackhpc_repo_rocky_9_highavailability_url }}"
     file: "rocky-addons.repo"
     tag: "highavailability"
+  - url: "{{ stackhpc_repo_rocky_9_sig_security_common_url }}"
+    file: "Rocky-SIG-Security-Common.repo"
+    tag: "sig_security_common"
```

Comment:
Does this match the `[name]` in the repo file?

Reply:
The upstream repo file looks like this:

```
[security-common]
name=Rocky Linux $releasever - SIG Security Common
mirrorlist=https://mirrors.rockylinux.org/mirrorlist?arch=$basearch&repo=rocky-sig-security-common-$releasever
#baseurl=http://dl.rockylinux.org/$sigcontentdir/$releasever/security/$basearch/security-common/
gpgcheck=1
enabled=1
countme=1
gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-Rocky-SIG-Security
```

Do we want to match it?

Reply:
We are removing this part for now and will come back to it later when building container images.